knative-operator/pkg/controller/knativeserving/consoleclidownload/consoleclidownload.go

```diff
@@ -64,31 +66,24 @@ func reconcileKnCCDResources(instance *servingv1alpha1.KnativeServing, apiclient
 	return nil
 }
 
-// Check for deployment and Route of kn ConsoleCLIDownload resources
+// Check for Knative Service and URL of kn ConsoleCLIDownload resources
 func checkResources(manifest *mf.Manifest, instance *servingv1alpha1.KnativeServing, api client.Client) error {
 	log.Info("Checking deployments")
-	for _, u := range manifest.Filter(mf.ByKind("Deployment"), mf.ByKind("Route")).Resources() {
-		switch u.GetKind() {
-		case "Deployment":
-			deployment := &appsv1.Deployment{}
-			err := api.Get(context.TODO(), client.ObjectKey{Namespace: u.GetNamespace(), Name: u.GetName()}, deployment)
-			if err != nil {
+	for _, u := range manifest.Filter(mf.ByGVK(servingv1.SchemeGroupVersion.WithKind("Service"))).Resources() {
```

Comment:
Do we even need to do this at this point? You're hardcoding the Service name below so we might get away with the same thing here maybe?

Reply:
Sounds like a great idea to me.